api.planx.uk/modules/send/utils/helpers.ts

```diff
@@ -38,21 +38,20 @@ export async function logPaymentStatus({
       });
     } catch (e) {
       reportError({
-        message: "Failed to insert a payment status",
-        error: e,
-        govUkResponse,
+        error: `Failed to insert a payment status: ${e}`,
+        context: { govUkResponse },
       });
     }
   }
 }
 
 // tmp explicit error handling
-export function reportError(obj: object) {
+export function reportError(report: { error: any, context: object }) {
   if (airbrake) {
-    airbrake.notify(obj);
+    airbrake.notify(report);
```

Comment:
This just accepts an `any`, but the actual type is documented here - https://github.com/airbrake/airbrake-js/tree/master/packages/browser#notice-annotations